state/etcd/schema.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package etcd
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
```

Comment:
Just noticed it. This is a weird package, not the official protobuf one. (And gopls seems to like to auto-complete this)

Please switch to the official protobuf then we should be good.

Reply:
Thanks  :+1: